### PR TITLE
change sign and repo to vscodium

### LINKED ISSUE #149: codium install error
```
 > [linux/amd64 13/16] RUN wget -qO - https://gitlab.com/paulcarroty/vscodium-deb-rpm-repo/raw/master/pub.gpg     | gpg --dearmor     | dd of=/usr/share/keyrings/vscodium-archive-keyring.gpg &&     echo 'deb [ signed-by=/usr/share/keyrings/vscodium-archive-keyring.gpg ] https://download.vscodium.com/debs vscodium main'     | tee /etc/apt/sources.list.d/vscodium.list &&     apt-get update -q &&     apt-get install -y codium &&     apt-get autoclean &&     apt-get autoremove &&     rm -rf /var/lib/apt/lists/*:
0.853 Get:15 http://archive.ubuntu.com/ubuntu jammy-updates/multiverse amd64 Packages [61.2 kB]
0.854 Get:16 http://archive.ubuntu.com/ubuntu jammy-updates/main amd64 Packages [1920 kB]
0.867 Get:17 http://archive.ubuntu.com/ubuntu jammy-updates/restricted amd64 Packages [2104 kB]
0.882 Get:18 http://archive.ubuntu.com/ubuntu jammy-updates/universe amd64 Packages [1358 kB]
0.901 Get:19 http://archive.ubuntu.com/ubuntu jammy-backports/main amd64 Packages [80.9 kB]
0.901 Get:20 http://archive.ubuntu.com/ubuntu jammy-backports/universe amd64 Packages [33.3 kB]
1.059 Get:21 https://ppa.launchpadcontent.net/mozillateam/ppa/ubuntu jammy/main amd64 Packages [43.5 kB]

4.235 E: Failed to fetch https://download.vscodium.com/debs/dists/vscodium/InRelease  Clearsigned file isn't valid, got 'NOSPLIT' (does the network require authentication?)
4.235 E: The repository 'https://download.vscodium.com/debs vscodium InRelease' is not signed.
```

https://github.com/Tiryoh/docker-ros2-desktop-vnc/actions/runs/8495664213/job/23272192701

Full log: [0_build-and-deploy.txt](https://github.com/Tiryoh/docker-ros2-desktop-vnc/files/14814537/0_build-and-deploy.txt)

### PR DESCRIPTION
**Pull request by sbarcelona11**
fix #149

### CODE REVIEW

File: foxy/Dockerfile [hunk 1]
```diff
@@ -76,10 +76,10 @@ RUN DEBIAN_FRONTEND=noninteractive add-apt-repository ppa:mozillateam/ppa -y &&
     rm -rf /var/lib/apt/lists/*
 
 # Install VSCodium
-RUN wget -qO - https://gitlab.com/paulcarroty/vscodium-deb-rpm-repo/raw/master/pub.gpg \
-    | gpg --dearmor \
+RUN wget https://gitlab.com/paulcarroty/vscodium-deb-rpm-repo/raw/master/pub.gpg \
+    -O /usr/share/keyrings/vscodium-archive-keyring.asc \
     | dd of=/usr/share/keyrings/vscodium-archive-keyring.gpg && \
```
[the remark this answers is not in the document]
Comment (Tiryoh) [reply foxy/Dockerfile:81]: Hi @sbarcelona11 , Thanks for the pull request.

Could you fix these lines to make it work?

**Review by Tiryoh — CHANGES_REQUESTED** [review]

File: humble/Dockerfile [hunk 2]
```diff
@@ -76,10 +76,10 @@ RUN DEBIAN_FRONTEND=noninteractive add-apt-repository ppa:mozillateam/ppa -y &&
     rm -rf /var/lib/apt/lists/*
 
 # Install VSCodium
-RUN wget -qO - https://gitlab.com/paulcarroty/vscodium-deb-rpm-repo/raw/master/pub.gpg \
-    | gpg --dearmor \
+RUN wget https://gitlab.com/paulcarroty/vscodium-deb-rpm-repo/raw/master/pub.gpg \
+    -O /usr/share/keyrings/vscodium-archive-keyring.asc \
     | dd of=/usr/share/keyrings/vscodium-archive-keyring.gpg && \
```
Comment (Tiryoh) [inline humble/Dockerfile:81]:
```suggestion
    -O /usr/share/keyrings/vscodium-archive-keyring.asc && \
```

File: iron/Dockerfile [hunk 3]
```diff
@@ -77,10 +77,10 @@ RUN DEBIAN_FRONTEND=noninteractive add-apt-repository ppa:mozillateam/ppa -y &&
     rm -rf /var/lib/apt/lists/*
 
 # Install VSCodium
-RUN wget -qO - https://gitlab.com/paulcarroty/vscodium-deb-rpm-repo/raw/master/pub.gpg \
-    | gpg --dearmor \
+RUN wget https://gitlab.com/paulcarroty/vscodium-deb-rpm-repo/raw/master/pub.gpg \
+    -O /usr/share/keyrings/vscodium-archive-keyring.asc \
     | dd of=/usr/share/keyrings/vscodium-archive-keyring.gpg && \
```
Comment (Tiryoh) [inline iron/Dockerfile:82]:
```suggestion
    -O /usr/share/keyrings/vscodium-archive-keyring.asc && \
```

File: rolling/Dockerfile [hunk 4]
```diff
@@ -77,10 +77,10 @@ RUN DEBIAN_FRONTEND=noninteractive add-apt-repository ppa:mozillateam/ppa -y &&
     rm -rf /var/lib/apt/lists/*
 
 # Install VSCodium
-RUN wget -qO - https://gitlab.com/paulcarroty/vscodium-deb-rpm-repo/raw/master/pub.gpg \
-    | gpg --dearmor \
+RUN wget https://gitlab.com/paulcarroty/vscodium-deb-rpm-repo/raw/master/pub.gpg \
+    -O /usr/share/keyrings/vscodium-archive-keyring.asc \
     | dd of=/usr/share/keyrings/vscodium-archive-keyring.gpg && \
```
Comment (Tiryoh) [inline rolling/Dockerfile:82]:
```suggestion
    -O /usr/share/keyrings/vscodium-archive-keyring.asc && \
```

File: foxy/Dockerfile (hunk 1 above)
[the remark this answers is not in the document]
Comment (sbarcelona11) [reply foxy/Dockerfile]: fixed

File: iron/Dockerfile (hunk 3 above)
Comment (sbarcelona11) [reply iron/Dockerfile]: fixed

File: humble/Dockerfile (hunk 2 above)
Comment (sbarcelona11) [reply humble/Dockerfile]: fixed

File: rolling/Dockerfile (hunk 4 above)
Comment (sbarcelona11) [reply rolling/Dockerfile]: fixed

**Comment by Tiryoh** [comment]
Hi @sbarcelona11, thanks for the fix.
I pushed a commit to fix some typos in the commit [412abf6](https://github.com/Tiryoh/docker-ros2-desktop-vnc/pull/148/commits/412abf61c840d67a77e2e9daef6c33b24b0ca278).

**Comment by sbarcelona11** [comment]
@Tiryoh Thanks to you! I'm using your image to work with ROS2 and Robotics. I would like you to create a version with GPU support. Can you?

**Pull request merged**